pubspec.yaml

```diff
@@ -12,12 +12,12 @@ dependencies:
   flutter:
     sdk: flutter
   # webview plugin
-  flutter_inappwebview: 6.0.0
+  flutter_inappwebview: ^6.0.0-beta.22
```

Comment:
should have dedicated branch for CNB.

Reply:
Yes, we have a `cnb_supported` branch specifically for CNB